README.md

```diff
@@ -75,6 +75,7 @@ if err != nil {
 The PIV applet has three unique credentials:
 
 * Management key (3DES key) used to generate new keys on the YubiKey.
+  * YubiKey firmware 5.4.0+ adds support for AES128/192/256 keys
```

Comment:
nit: indent

Reply:
This was done deliberately as something specific for management keys

![Screenshot 2024-07-08 at 15 48 29](https://github.com/go-piv/piv-go/assets/673216/82c10728-49ba-4e04-ad24-4b793e9c24f4)

Reply:
Replaced 2 space indent with tab